src/main/java/crazypants/enderio/conduit/item/filter/ItemFilterLimited.java

```diff
@@ -0,0 +1,253 @@
+import crazypants.enderio.conduit.item.NetworkedInventory;
+
+public class ItemFilterLimited extends ItemFilter {
+
+    public ItemFilterLimited() {
+        this(10, true);
+    }
+
+    public ItemFilterLimited(int numItems, boolean isAdvanced) {
+        super(numItems, isAdvanced);
+    }
+
+    @Override
+    public String getInventoryName() {
+        return "Limited Item Filter";
+    }
+
+    @Override
+    @SideOnly(Side.CLIENT)
+    public IItemFilterGui getGui(GuiExternalConnection gui, IItemConduit itemConduit, boolean isInput) {
+        ItemConduitFilterContainer cont = new ItemConduitFilterContainer(itemConduit, gui.getDir(), isInput);
+        LimitedItemFilterGui limitedItemFilterGui = new LimitedItemFilterGui(gui, cont, !isInput, isInput);
+        limitedItemFilterGui.createFilterSlots();
+        return limitedItemFilterGui;
+    }
+
+    @Override
+    public void createGhostSlots(List<GhostSlot> slots, int xOffset, int yOffset, Runnable cb) {
+        int index = 0;
+        int numRows = (int) (double) (items.length / 5);
+        for (int row = 0; row < numRows; ++row) {
+            for (int col = 0; col < 5; ++col) {
+                int x = xOffset + col * 18;
+                int y = yOffset + row * 20;
+                slots.add(new ItemFilterGhostSlot(index, x, y, cb));
+                index++;
+            }
+        }
+    }
+
+    @Override
+    public int getMaxCountThatCanPassFilter(ItemStack item) {
+        if (item == null || item.getItem() == null) {
+            return 0;
+        }
+        int count = 0;
+        for (int i = 0; i < items.length; i++) {
+            ItemStack invStack = items[i];
+            if (invStack != null && itemStackMatched(invStack, item, i)) {
+                count += invStack.stackSize;
+            }
+        }
+        return count;
+    }
+
+    @Override
+    public boolean isLimited() {
+        return true;
+    }
+
+    public int getInsertLimit(ISidedInventory inv, ItemStack item) {
+        if (item == null || item.getItem() == null) {
+            return 0;
+        }
+        int count = 0;
+        for (int i = 0; i < inv.getSizeInventory(); i++) {
+            ItemStack invStack = inv.getStackInSlot(i);
+            if (invStack != null && itemStackMatched(invStack, item, i)) {
+                count += invStack.stackSize;
+            }
+        }
+        return getMaxCountThatCanPassFilter(item) - count;
+    }
+
+    public int getInsertLimitInv(ISidedInventory inv, ItemStack item) {
+        if (item == null || item.getItem() == null) {
+            return 0;
+        }
+        int count = 0;
+        for (int i = 0; i < inv.getSizeInventory(); i++) {
+            ItemStack invStack = inv.getStackInSlot(i);
+            if (invStack != null && itemStackMatched(invStack, item, i)) {
+                count += invStack.stackSize;
+            }
+        }
+        return count - getMaxCountThatCanPassFilter(item);
```

Comment:
It can be simplified (if I didn't miss anything):
```suggestion
        return -getInsertLimit(inv, item);
```